src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Members.cs

```diff
@@ -767,9 +767,23 @@ public override void VisitParameter(IParameterSymbol symbol)
 
             if (includeType)
             {
+                if (symbol.IsRefScoped &&
+                    format.ParameterOptions.IncludesOption(SymbolDisplayParameterOptions.IncludeScoped))
```

Comment:
>format.ParameterOptions.IncludesOption(SymbolDisplayParameterOptions.IncludeScoped) [](http://example.com/codeflow?start=20&length=83)

`ref` is added conditionally. Perhaps we should also the option that controls that.